support/cas-server-support-pm-webflow/src/main/java/org/apereo/cas/pm/web/flow/PasswordManagementWebflowConfigurer.java

```diff
@@ -112,6 +117,11 @@ private void createPasswordResetFlow() {
             final ActionState initializeLoginFormState = getState(flow, CasWebflowConstants.STATE_ID_INIT_LOGIN_FORM, ActionState.class);
             final String originalTargetState = initializeLoginFormState.getTransition(CasWebflowConstants.STATE_ID_SUCCESS).getTargetStateId();
             final SubflowState pswdResetSubFlowState = createSubflowState(flow, CasWebflowConstants.STATE_ID_PASSWORD_RESET_SUBFLOW, FLOW_ID_PASSWORD_RESET);
+
+            flow.getTransitionableState(CasWebflowConstants.STATE_ID_REAL_SUBMIT).getEntryActionList()
```

Comment:
Switch to `getTransitionableState` please.

Reply:
Good catch. I didn't notice there was a member for that. I switched the ones from my past changes as well...